cfme/roles.py

```diff
@@ -895,3 +895,19 @@ def _remove_from_all(roles, r_page):
         }
     }
 }
+
+
+ROLE_ACCESS_UI_510Z = list(
```

Comment:
Sorry for the confusion, but I meant that the existing `role_access_ui_510z` and `role_access_ui_511z` should be changed to all caps. But these lists should still be in the test file.